src/shim.js

```diff
@@ -97,7 +91,28 @@ const commands = {
         completeCommand(command);
     },
 
-    noop: command => completeCommand(command)
+    noop: command => completeCommand(command),
+    
+    invokeAsyncMethod: function (command) {
+        let target = objectSpace[command.target];
+        target[command.params[0]].apply(target, command.params.slice(1).concat(result => {
```

Comment:
Throw error here if target doesn't exist. This would never happen, but it would be good to say can't find target object to the user.